on-prem/authentication/introduction.md

```diff
@@ -0,0 +1,13 @@
+# Introduction
```

Comment:
Add a section about enabling/disabling? I think that would go on this page

Reply:
Also a BIG FAT WARNING saying that if they disable the builtin username + password provider, and don't have proper setup for another provider they WILL NOT BE ABLE TO ACCESS THE PRODUCT. They would have to go into the DB and enable it again